utils/helpers/mergeDeep.cjs

```diff
@@ -0,0 +1,25 @@
+const isObject = (variable) => {
+  return typeof variable === 'object' && !Array.isArray(variable) && variable !== null;
```

Comment:
```suggestion
  return `${{}}`;
```

Jouw checks zijn explicieter en duidelijker om de intentie van de functie te bepalen (als je leest), maar dit is ook duidelijk zat (voor ES6+ kenners). Wat vind jij er van?

Reply:
Ik snap het niet helemaal, de ene returned een `boolean`, de andere een `string` met "[object object]"